client/blocks/reader-related-card-v2/index.jsx

```diff
@@ -27,9 +26,9 @@ function FeaturedImage( { image, href } ) {
 		} } ></div> );
 }
 
-function AuthorAndSiteFollow( { post, site } ) {
+function AuthorAndSiteFollow( { post } ) {
 	const siteUrl = getStreamUrl( post.feed_ID, post.site_ID );
-	const authorAndSiteAreDifferent = site.title.toLowerCase() !== post.author.name.toLowerCase();
+	const authorAndSiteAreDifferent = post.site_name.toLowerCase() !== post.author.name.toLowerCase();
```

Comment:
yeah, though `title` is ... fancy. If the site is missing an actual title, it falls back to the domain. We lost that with this change. Hrm.